src/compiler/types.ts

```diff
@@ -5220,6 +5220,12 @@ namespace ts {
         IsNeverIntersectionComputed = 1 << 26, // IsNeverLike flag has been computed
         /* @internal */
         IsNeverIntersection = 1 << 27, // Intersection reduces to never
+
+        // Flags that require TypeFlags.Reference

```

Comment:
There is no `TypeFlags.Reference`. Move them to the section that requires `TypeFlags.Object` and use bits 25 and 26.